SU-SWS/jumpstart_ui
# Updated Decanter to latest V6.

# READY FOR REVIEW

# Summary
- Updates Decanter to latest V6 and fixes the API breaking changes.

# Needed By (Date)
- Sooner the better or you may get broken builds

# Urgency
- Yep, please soon.

# Steps to Test

1. Check out this branch
2. run `nvm use`
3. run `npm run publish`
4. Review git for diff (should be none)
5. For bonus points, review on a live build that everything still looks good.

# Affected Projects or Products
- All the things

# Associated Issues and/or People
- 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
